src/braket/default_simulator/state_vector_simulation.py

```diff
@@ -66,6 +66,12 @@ def __init__(self, qubit_count: int, shots: int, batch_size: int):
         self._post_observables = None
 
     def evolve(self, operations: List[GateOperation]) -> None:
+        """
+        Evolve simulation by applying the provided operations.
+
+        Args:
+            operations (List[GateOperation]): Operations to apply.
+        """
```

Comment:
Are this and `expectation` docstring really necessary, or does this inherit documentation from a superclass? If the former, nice catch and thanks for adding!

Reply:
Ah good call, didn't realize this